portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact(){
```

Comment:
I like this function! Makes the page interesting

Reply:
Thanks